src/app/main/employees-sheet/sheets-list/sheets-list.component.ts

```diff
@@ -57,4 +61,11 @@ export class SheetsListComponent implements OnInit {
   })
   }
 
+  public startSync(sheet: AttendanceSheet){
+    this.fedenaSyncService.syncAttendanceSheet({employees_attendance_id: sheet.id}).subscribe(response => {
+      sheet.state = "syncing"
```

Comment:
We don't need to notify the user that the sync started?

Reply:
I think we need another message for syncing the attendance sheet to Fedena but let's keep it like this for now